nrfcloud/settings.ts

```diff
@@ -67,6 +72,11 @@ export const getSettings = ({
 			accountDevicePrivateKey,
 			accountDeviceClientId,
 			mqttTopicPrefix,
+			coapEndpoint:
+				coapEndpoint === undefined
+					? defaultCoAPEndpoint
+					: new URL(coapEndpoint),
+			coapPort: Number(coapPort ?? `5684`),
```

Comment:
```suggestion
			coapPort: parseInt(coapPort ?? `5684`, 10),
```

Port may only be a integer.